src/stylesheets/core/_functions.scss

```diff
@@ -106,7 +106,10 @@ Leaves bools as is
     @return quote($value);
   }
   @if type-of($value) == 'number' {
-    @if unit($value) == 'xl' or unit($value) == 'xs' {
+    @if unit($value) == 'px' {
```

Comment:
Looks like the `if` and `elseif` can be combined into one statement here since they are returning the same value. 
```
@if unit($value) == 'px' or unit($value) == 'xl' or unit($value) == 'xs' {
```
Might be able to you `index` also (not tested):
```
@if index(('px', 'xl', 'xs'), unit($value)) {
```

Reply:
Are there any drawbacks to quoting all `px` values?

Reply:
I _do_ wonder if there's a more succinct way to write this! Curious about that index technique

Reply:
Re drawbacks: _probably_ not, particularly in the long term. It could be a short-term issue and throw an error if some older USWDS custom value were being passed to the spacing function. But this seems like a pretty small slice, and one that we'd probably want to iron out by 2.0 anyway

Reply:
It _might_ be an issue for some other custom implementations, too — but that also feels like a stretch

Reply:
I'm gonna try that index technique (it worked!) and re-commit.